### PR TITLE
Last batch of changes

### PR DESCRIPTION
**Pull request by jeromekelleher**
OK, I think this is it for the changes I'd like to make.

There is potential controversy in the PopGen model section, so please do go through that carefully and make sure you agree with what we're saying now!

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -295,17 +315,18 @@ \subsection*{Human Data Model}
...
 Structurally, the HDM encourages human understanding
 by avoiding redundancy in the description where possible and by providing a
 mechanism for specifying default values that are inherited hierarchically.
 For values that repeat
-across fields, defaults may be used to implicitly assign default values to
+across fields, the ``defaults'' mechanism may be used to implicitly assign default values to
 fields of the given type.  A default is superseded by an explicitly provided
 value if given. Size values are inherited naturally following the
+the document. Size values are inherited naturally following the
 progression of time. For example, if an epoch \texttt{start\_size} is not
```
Comment (grahamgower) [inline paper.tex]: Copy/paste issue here?

Comment (jeromekelleher) [reply paper.tex]: Oops! Should be fixed now.

**Review by apragsdale — APPROVED** [review]
Very good! Thanks @jeromekelleher

File: response-to-reviewers.tex [hunk 2]
```diff
@@ -560,12 +576,10 @@ \subsection*{Random thoughts I had while reading the paper, to discuss (or
...
-TODO
-
-\ggcomment{
-Say something about sparse versus dense.
-Then refer to our docs so that we don't have to copy/paste the docs in the manuscript.
-}
+YAML supports a similar type of shorthand, and there are various
+examples in the tutorial where we use it to increase legibility,
+e.g.
+\url{https://popsim-consortium.github.io/demes-spec-docs/main/tutorial.html#exponential-size-changes}
```
Comment (grahamgower) [inline response-to-reviewers.tex:582]: It seems you've interpreted Laurent's point here to refer to flow-style versus block-style YAML? I interpreted it more about using a matrix `M` to represent migrations (a dense representation) versus what we do where a source/dest migration list element refers to `M[source][dest]` (a sparse representation).

**Review by grahamgower — APPROVED** [review]
Thanks @jeromekelleher, lots clearer now. It's getting a bit hard for me to see things from the diff now that so much has been shuffled, but I vote commit this and I'll do another pass over the full documents in the next few days to double check I didn't miss anything.

File: response-to-reviewers.tex (hunk 2 above)
Comment (jeromekelleher) [reply response-to-reviewers.tex:582]: Yes, that's true. He sort of implied both I think, so I went with the version that's practical?

Feel free to update in a follow up, I don't have  strong opinions here.

**Comment by jeromekelleher** [comment]
OK, great. Let's just wait for @molpopgen to give his thumbs-up so, merge, and then we can send the current version (and diff) to other authors for approval on Slack? 

(I probably won't have time to do this as I'm away for the weekend tomorrow - would you mind doing this once you've had a look over @grahamgower?)

**Comment by grahamgower** [comment]
Yep, can do.

File: response-to-reviewers.tex [hunk 3]
```diff
@@ -536,21 +542,31 @@ \subsection*{Random thoughts I had while reading the paper, to discuss (or
...
+
+Thus, from msprime's perpective, there is no difference between what we
+have currently (the HDM converted to MDM via the parser) and what
+you propose (the MDM validated by the parser).
+
+If the HDM did not exist, we would still require parsers because validating
+the full MDM is a lengthy and tedious task.
+A key goal that we set was for the ecosystem to be ``batteries-included''
+from the outset, which is why we have implemented several high-quality
+parsers (Python, and C with Rust, Julia in development) which should
```
Comment (molpopgen) [inline response-to-reviewers.tex:566]: The rust parser is out and ready for use.

**Review by molpopgen — COMMENTED** [review]
I think this is great.  The changes to the model section are good, I think.  No substantive comments on my end.

File: response-to-reviewers.tex (hunk 3 above)
Comment (apragsdale) [reply response-to-reviewers.tex:566]: I'll take care of this on my round of edits!

**Pull request merged**